RevenueCatUI/Templates/Components/Stack/StackComponentView.swift

```diff
@@ -56,6 +56,10 @@ struct StackComponentView: View {
         .background(viewModel.backgroundColor)
         .cornerBorder(border: viewModel.border,
                       radiuses: viewModel.cornerRadiuses)
+        .applyIfLet(viewModel.shadow) { view, shadow in
+            // Without compositingGroup(), the shadow is applied to the stack's children as well.
+            view.compositingGroup().shadow(shadow: shadow)
```

Comment:
The reason for not adding `compositingGroup()` directly to the new `shadow()` modifier is that I wanted the latter to behave the same as the built-in one to avoid confusion. Let me know if you disagree!

Reply:
TIL about `compositingGroup()` 😅